libs/network/network.c

```diff
@@ -103,7 +114,17 @@ JsVar *networkGetAddressAsString(unsigned char *ip, int nBytes, unsigned int bas
   return jsvNewFromString(data);
 }
 
-void networkPutAddressAsString(JsVar *object, const char *name,  unsigned char *ip, int nBytes, unsigned int base, char separator) {
+/**
+ * Put an IP address as a child property of a JsVar.
```

Comment:
Just to note that this (and others) aren't just for IP addresses. They're used for MACs too...